Manito/Manito/Screens/Letter/LetterViewController.swift

```diff
@@ -134,6 +158,10 @@ final class LetterViewController: BaseViewController {
                 $0.leading.trailing.bottom.equalTo(view.safeAreaLayoutGuide)
             }
         }
+        view.addSubview(emptyLabel)
+        emptyLabel.snp.makeConstraints {
+            $0.centerX.centerY.equalToSuperview()
```

Comment:
p5;

```suggestion
            $0.center.equalToSuperview()
```
이렇게 쓸 수 있겠군요

Reply:
수정완욜

[5302c54](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/258/commits/5302c540fff53af822cbda3ee491171ed28df6b8)

듀나짱짱